pom.xml

```diff
@@ -237,10 +237,7 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>2.10.4</version>
                 <configuration>
-                    <reportOutputDirectory>${project.reporting.outputDirectory}/apidocs
-                    </reportOutputDirectory>
-                    <destDir>apidocs</destDir>
-                    <nohelp>true</nohelp>
```

Comment:
In [Apache Maven Javadoc Plugin](http://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html) reference:
<dl>
  <dt>reportOutputDirectory</dt>
  <dd>Default value is: ${project.reporting.outputDirectory}/apidocs</dd>
  
  <dt>destDir</dt>
  <dd>Default value is: apidocs</dd>
  
  <dt>nohelp</dt>
  <dd>Default value is: false → I don't see any reason to exclude help page.</dd>  
</dl>


So we can safely omit these parameters in this `pom.xml`.